### ISSUE TITLE
Please provide a nullsafety version of your library

### ISSUE THREAD

**Issue by sma** [johnpryan/shelf_gzip#3]
All you need to change is `pubspec.yaml` like so:

```
name: shelf_gzip
description: Shelf middleware to GZIP responses
version: 4.0.0
author: John Ryan <john.p.ryan4@gmail.com>
homepage: https://github.com/johnpryan/shelf_gzip

environment:
  sdk: '>=2.12.0-0 <3.0.0'

dependencies:
  shelf: ">=1.0.0 <2.0.0"
```

Currently, your library doesn't work with the latest version of `shelf`.

**Comment by gmpassos** [comment]
FYI:
https://github.com/johnpryan/shelf_gzip/pulls

**Comment by gmpassos** [comment]
Resolved with ver 4.0.0
